DotnetMakeDeb/Deb/DebPackage.cs

```diff
@@ -279,6 +280,31 @@ public void ReadSpecification(string fileName)
 						continue;
 					}
 
+					m = Regex.Match(line, @"^contfile\s*:\s*(\S+)\s+(\S+)(?:\s+(text))?$", RegexOptions.IgnoreCase);

```

Comment:
Your suggested documentation (not part of the PR) mentioned something abour file modes. This isn't supported by this regex.

Reply:
Oh, I do not notice this. I am not well versed in regex, that is why I just copied your code for the regex.

Reply:
The main point of a regex is that it contains certain placeholders that can match certain input. This understanding should be sufficient for this case. Anything in round parentheses is captured, i.e. can be accessed in the returned match variable by its index. There are simply no capturing placeholders for the extra values in this regex. It can only accept 2 paths and the optional "text" keyword. I guess you copied the regex from one of the script files but the `file` directive is probably better here.

Reply:
Oh @ygoe actually I do not plan to use file modes for this one, I will revise the documentation instead.

Reply:
Now it should no longer become a problem because user can define their own `Mode` for the control file.

Reply:
Currently I am using following regex (customized version from what you have made for `file`):
```
^controlfile\s*:\s*(\S+)(?:\s+(text))?(?:\s+([0-9]+)(?:\s+([0-9]+)\s+([0-9]+))?)?\s*$
```
which able to match with:
1. SourcePath
2. IsText
3. Mode
4. UserId
5. GroupId

I have tested it with https://regex101.com/ as well.